tests/e2e/test_rosacli_ingress.go

```diff
@@ -178,6 +179,41 @@ var _ = Describe("Edit default ingress",
 				ingress, _ = defaultIngress(*ingressList)
 				Expect(ingress.LBType).Should(ContainSubstring(ingress.LBType))
 			})
+
+		It("can customize ingress controller at install - [id:65798]",
+			labels.High,
+			labels.Day1Post,
+			labels.NonHCPCluster,
+			func() {
+				if isHosted {
+					Skip("This case is for standard ROSA clusters only")
+				}
+				if !PH.LoadProfileYamlFileByENV().ClusterConfig.IngressCustomized {
+					Skip("The ingress must be customized at install")
+				}
+				output, err := ingressService.ListIngress(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				ingressList, err := ingressService.ReflectIngressList(output)
+				Expect(err).ToNot(HaveOccurred())
+				defaultIngress := func(ingressList rosacli.IngressList) (*rosacli.Ingress, bool) {
+					for _, ingress := range ingressList.Ingresses {
+						if ingress.Default == "yes" {
+							return &ingress, true
+						}
+					}
+					return nil, false
+				}
+
+				ingress, _ := defaultIngress(*ingressList)
```

Comment:
Before call `ingress`, need to do `Expect(ingress).NotTo(BeNil())` or `Expect(existed).To(BeTrue())`